Packet++/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ add_library(
   src/TLVData.cpp
   src/TpktLayer.cpp
   src/UdpLayer.cpp
+  src/InfiniBandLayer.cpp
```

Comment:
nit: the files here are order alphabetically

Reply:
@egecetin btw, should be able to catch this in your CI PR?

Reply:
@tigercosmos You mean with gersemi?

Reply:
ok, have fix it.